### PR TITLE
Update requirements-txt files to reflect latest dependency versions

### PR DESCRIPTION
**Pull request by piiq**
This requirements-txt update should resolve the `aiohttp` vulnerability

### CODE REVIEW

File: requirements-full.txt [hunk 1]
```diff
@@ -7,42 +7,42 @@ anyio==3.4.0; python_full_version >= "3.6.2" and python_version >= "3.6"
...
-cffi==1.15.0; implementation_name == "pypy" and python_version >= "3.6"
+certifi==2021.10.8; python_full_version >= "3.7.1" and python_version >= "3.6" and python_full_version < "4.0.0" and (python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.6.0") and (python_version >= "3.8" and python_full_version < "3.0.0" and python_version < "4.0" or python_version >= "3.8" and python_version < "4.0" and python_full_version >= "3.6.0")
+cffi==1.15.0; implementation_name == "pypy" and python_version >= "3.7" and python_full_version >= "3.6.1"
 chardet==4.0.0; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version >= "3.6"
-charset-normalizer==2.0.7; python_full_version >= "3.6.0" and python_version >= "3"
+charset-normalizer==2.0.8; python_full_version >= "3.7.1" and python_version >= "3.6" and python_full_version < "4.0.0" and (python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.6.0") and (python_version >= "3.8" and python_full_version < "3.0.0" and python_version < "4.0" or python_version >= "3.8" and python_version < "4.0" and python_full_version >= "3.6.0")
```
Comment (aia) [inline requirements-full.txt:29]: You will want to bump charset-normalizer in https://github.com/GamestonkTerminal/GamestonkTerminal/blob/main/build/conda/conda-3-8-env-full.yaml#L13

Comment (piiq) [reply requirements-full.txt:29]: thanks

**Review by Chavithra — APPROVED** [review]
Seems to work, I was able to :
- install with poetry
- run terminal.py and display candle graph
- run the tests

The `conda env`seems also to build properly in actions.

**Review by aia — APPROVED** [review]

**Comment by piiq** [comment]
> Should we change `pylint` version from `2.11.1` to `2.12.1` in workflows ?

Yes, but I'd leave it to the next dependency update PR. This PR was started specifically to solve the dependabot alert

**Pull request merged**